tarantool/schema.py

```diff
@@ -26,8 +26,11 @@ def __init__(self, index_row, space):
         self.unique = index_row[4]
         self.parts = []
         if isinstance(index_row[5], (list, tuple)):
-            for k, v in index_row[5]:
-                self.parts.append((k, v))
+            for part in range(len(index_row[5])):
```

Comment:
Please, add tests for this part of code

Reply:
OK. But, I've changed format of parts to dict `{'field': part[0], 'type': part[1]}`, because for me parts doesn't matter. I don't use them. But this change brokes backward compatibility. So, I should do this by older way? I mean like this:
```
if isinstance(part, list):
    self.parts.append((part[0], part[1]))
else:
    self.parts.append((part['field'], part['type']))
```